portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,11 @@
   width: 650px;
 }
 
-#greeting-container {
+#voting-method-container {
   margin-top: 20px;
 }
+
+img.resize {
```

Comment:
From https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming:

Instead of presentational or cryptic names, always use ID and class names that reflect the purpose of the element in question, or that are otherwise generic.